rapid_response_xblock/block.py

```diff
@@ -89,6 +89,7 @@ def student_view_aside(self, block, context=None):  # pylint: disable=unused-arg
         )
         fragment.add_css(get_resource_bytes("static/css/rapid.css"))
         fragment.add_javascript(get_resource_bytes("static/js/src/rapid.js"))
+        fragment.add_javascript(get_resource_bytes("static/js/src/d3.v4.min.js"))
```

Comment:
I think it makes more sense to move this to `static/js/lib`. Also it seems like we aren't using the jquery file in that directory so that can be deleted.

Reply:
Sure, but where's the jquery file?

Reply:
Heh... looks like it's not even checked in. I have it in my local repo, but I'm assuming it didn't make it into any of my commits because of [this gitignore rule](https://github.com/mitodl/rapid-response-xblock/blob/master/.gitignore#L18)